src/jira.js

```diff
@@ -303,7 +303,7 @@ export default class JiraApi {
 
       return response;
     } catch (e) {
-      throw new Error(e);
+      throw new Error(e.errorMessages.join(', '));
```

Comment:
My concern with this is it could mute any errors that show up under "errors". See the docs here: https://developer.atlassian.com/server/jira/platform/rest-apis/#error-responses

Were you not able to see the error messages in the log at all?

I'm wondering if they were being displayed as `[object, object]` because how nested it is.

Reply:
Good point - I subsequently noticed that there was an **errors** property too. So the issue is that the error thrown is supposed to receive a string, not an object - so the current implementation only shows `[Object object]` as you say. Would you like me to modify the pr to:

- join all `errorMessages` and `error` values OR
- `JSON.stringify` the object?

Reply:
`JSON.stringify` would work great! I was going to say to wrap stringify in a try/catch, but as it turns out [stringify rarely returns an error](https://thecodebarbarian.com/the-80-20-guide-to-json-stringify-in-javascript#errors-and-edge-cases).

Would you mind adding [a small test too](https://github.com/jira-node/node-jira-client/blob/master/test/jira-tests.js)?